content/en/ecosystem/vendors.md

```diff
@@ -38,6 +38,7 @@ OpenTelemetry in their commercial products.
 | observIQ                   | Yes               | Yes         | [observiq.com/...](https://docs.bindplane.observiq.com)                                                                                               |
 | Promscale                  | No                | Yes         | [timescale.com/promscale](https://www.timescale.com/promscale)                                                                                        |
 | Sentry Software            | Yes               | Yes         | [sentrysoftware.com/...](https://www.sentrysoftware.com/products/hardware-sentry-opentelemetry-collector.html)                                        |
+| ServicePilot               | No                | Yes         | [servicepilot.com](https://www.servicepilot.com/en/doc/apm/#opentelemetry)                                                                                                                        |
```

Comment:
there seems to be an issue with the table formatting and because of that the build is failing, please review the netlify logs to see what's going wrong